src/d/actor/d_a_sail.cpp

```diff
@@ -438,8 +456,39 @@ u8 l_AmastDL[] ALIGN_DECL(32) = {
 };
 
 /* 000000EC-00000364       .text setCorrectNrmAngle__15daSail_packet_cFsf */
-void daSail_packet_c::setCorrectNrmAngle(short, float) {
-    /* Nonmatching */
+void daSail_packet_c::setCorrectNrmAngle(s16 param_0, f32 param_1) {
+    s16 rnd = 900 + (s32)cM_rndF(200.0f);
+    this->m1C38 += rnd;
```

Comment:
is the temp variable necessary for the match? or does this work too
```suggestion
    this->m1C38 += 900 + (s32)cM_rndF(200.0f);
```